sample/src/main/java/com/clevertap/demo/ctvaultsdk/MainActivity.kt

```
@@ -0,0 +1,957 @@
+
+    private fun parseInputValue(input: String): Any? {
+        return try {
+            when (selectedDataType) {
+                DataType.STRING -> input
+                DataType.INT -> input.toInt()
+                DataType.LONG -> input.toLong()
+                DataType.FLOAT -> input.toFloat()
+                DataType.DOUBLE -> input.toDouble()
+                DataType.BOOLEAN -> input.toBoolean()
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)